code/modules/clothing/clothing.dm

```diff
@@ -366,6 +366,12 @@
 		else
 			to_chat(M, span_warning("[src] start[p_s()] to fall apart!"))
 
+// you just dont get the same feeling with handwashed clothes
+/obj/item/clothing/machine_wash()
+	. = ..()
+	AddComponent(/datum/component/onwear_mood, 2 MINUTES, /datum/mood_event/fresh_laundry, \
```

Comment:
Make these used named parameters, and if you're going to multiline it, do it like this:

```dm
AddComponent( \
    /datum/component/onwear_mood, \
    whatever = 2 MINUTES, \
    whatever = /datum/mood_event/fresh_laundry, \
    whatever = "etc", /* notice the trailing comma */ \
)